src/robot/handler/protobuf.py

```diff
@@ -24,12 +27,16 @@ def __init__(self, match_action_controller: MatchActionController):
         type_msg = bus_message.WhichOneof("message_content")
         if type_msg == "heartbeat":
             pass
+        elif type_msg == "movementEnded":
+            self.localization_controller.movement_done()
```

Comment:
also pass the "blocked" parameter of the movement_done message? even if it is not used yet

Reply:
Yes, done